cdc/async/async_initializer.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2024 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package async
+
+import (
+	"context"
+	"sync"
+
+	"github.com/pingcap/log"
+	"github.com/pingcap/tiflow/pkg/errors"
+	"github.com/pingcap/tiflow/pkg/workerpool"
+	"go.uber.org/atomic"
+	"go.uber.org/zap"
+)
+
+// Initializer is a helper struct to initialize a changefeed asynchronously.
+type Initializer struct {
```

Comment:
IMHO, the name of variables can be simplified further. the prefix `initializ` is spread all over the file, hard to read.